src/clean.js

```diff
@@ -34,6 +34,14 @@ function clean(node, newObj) {
     delete newObj.isDoubleQuote;
   }
 
+  if (node.kind === "inline") {
+    if (node.value.includes("___PSEUDO_INLINE_PLACEHOLDER___")) {
+      return null;
+    }
+
+    newObj.value = newObj.value.replace(/\r\n?|\n/g, "");
+  }
```

Comment:
Why do we need this?

Reply:
@czosel workaround in `newlines` for `inline` nodes, we can fix it after implement better ast for inline

Reply:
Alright! Was just a bit confused because this seems unrelated to the test util refactor.